### ISSUE TITLE
Keyboard accessibility 

### ISSUE THREAD

**Issue by ajanec01** [craftcms/cms#7415]
### Overview of keyboard accessibility issues in the Assets and the Category section

- Only one of the folders rin the navigation panel eceives keyboard focus, a click on either of them adds `tabindex` 
- Customise link is not keyboard accessible (missing href)- should be a button
![customise](https://user-images.githubusercontent.com/47136246/104253729-1d9e1580-546d-11eb-9e75-1bc48f7dc711.png)
- Not possible to reorder checkboxes in the customise dialog using keyboard only
- The clear search button not keyboard accessible and is missing the acc role
-  Next page and previous page are not keyboard accessible, they also need an accessible role  (buttons that switch the table view)
- Additional information  in the edit pop-up (triggered by the info icon)  is not keyboard accessible; other issues discussed in #7308  also apply 
![Edit-popup](https://user-images.githubusercontent.com/47136246/104254149-175c6900-546e-11eb-8759-44793aa988c2.png)
- Not possible to select and reorder folders using the keyboard only (customise dialog)
![reorder-folders](https://user-images.githubusercontent.com/47136246/104254373-8fc32a00-546e-11eb-8e71-dbfcdb5878b8.png)
- Delete heading is not keyboard accessible
- Preview and Edit options are not keyboard accessible; they are visible only on mouse hover (`opacity ` on the parent container); also nrv issue
![preview-edit](https://user-images.githubusercontent.com/47136246/104254476-cd27b780-546e-11eb-9306-718d32a09087.png)

### Additional info

- Craft version: 3.5.18

**Comment by ajanec01** [comment]
With the exception of the preview/edit buttons and additional information in the edit pop-up, all other issues appear on the Users page too.

**Comment by gcamacho079** [comment]
I’m closing this issue as many issues have been addressed as of 5.2.8, and some are stale due to UI changes.

## Resolved
- Source navigation has been updated so that all buttons are in the tab order, versus there only being one in the tab order and navigating via arrow keys
- The “Customize” link has been changed to a `button`
- The clear search button is keyboard-accessible and has an appropriate NRV
- Next and previous pagination buttons are keyboard-accessible and have appropriate NRV
- The “Delete heading” button is keyboard accessible
- Preview and Edit buttons are keyboard accessible and have appropriate NRV

## Outstanding
The primary outstanding issue from this ticket — reorder buttons not having appropriate NRV — has been moved into our issue tracking.
